builds/android/app/src/main/java/org/easyrpg/player/player/SafFile.java

```diff
@@ -150,31 +141,25 @@ public int createOutputFileDescriptor(boolean append) {
     }
 
     DirectoryTree getDirectoryContent() {
-        if (root == null || !root.isDirectory()) {
-            return null;
-        }
-
-        Uri root_uri = root.getUri();
-
         ArrayList<String> files = new ArrayList<>();
         ArrayList<Boolean> is_dir = new ArrayList<>();
 
         final ContentResolver resolver = context.getContentResolver();
-        final Uri childrenUri = DocumentsContract.buildChildDocumentsUriUsingTree(root_uri, DocumentsContract.getDocumentId(root_uri));
+        String documentId = DocumentsContract.getTreeDocumentId(rootUri);
+        if (DocumentsContract.isDocumentUri(context, rootUri)) {
+            documentId = DocumentsContract.getDocumentId(rootUri);
+        }
```

Comment:
no idea why this is needed. I copied this from the ``DocumentFile`` constructor. When its missing it doesn't work :shrug: